wp-content/themes/core/components/tabs/tabs.php

```diff
@@ -0,0 +1,39 @@
+<?php
+declare( strict_types=1 );
+
+/**
+ * @var array $args Arguments passed to the template
+ */
+// phpcs:ignore VariableAnalysis.CodeAnalysis.VariableAnalysis.UndefinedVariable
+$c = \Tribe\Project\Templates\Components\tabs\Controller::factory( $args );
+
+if ( empty( $c->tabs ) ) {
+	return;
+}
+
+?>
+
+<div <?php echo $c->get_classes(); ?> <?php echo $c->get_attrs(); ?>>
+	<div class="c-tabs__tablist-wrapper">
+
+		<?php if ( $c->layout === $c::LAYOUT_VERTICAL ) {
```

Comment:
I'm confused about this. If it's vertical we display a single button? Is this for mobile support? And why does it grab the first button label? What about the others?

Reply:
For the vertical layout, on mobile, the tablist buttons are collapsed/hidden. This button toggles the collapsed list of tabs similar to how we did it over on the [Conroe Resources](https://cidstagedev.wpengine.com/conroe-new-panels-2020-05/?secret=1) panel.  It's definitely a little weird to read at first, but the toggle button shows the currently selected tab's label. Wen a user chooses a different tab, the toggle label changes to match the newly selected tab. The actual tablist button options still exist within the drop-down and are used for changing the selected tab. There's a :poop:-ton of accessibility related to this pattern, hence the complexity for it.

For [the horizontal layout](https://www.figma.com/file/Wj2g7HVyBQWYIuxgNKMCxf/SQ1-Theme?node-id=2551%3A3215), the tabs just overflow: scroll horizontally.